dev/docker/doris/start.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Copyright 2024 Datastrato Pvt Ltd.
+# This software is licensed under the Apache License version 2.
+#
+
+DORIS_HOME="/opt/apache-doris"
+DORIS_FE_HOME="${DORIS_HOME}/fe/"
+DORIS_BE_HOME="${DORIS_HOME}/be/"
+
+# comment a code snippet about max_map_count, it's not necessary for IT environment
+
+DORIS_BE_SCRIPT="${DORIS_BE_HOME}/bin/start_be.sh"
+
+sed -i '/Please set vm.max_map_count/,/exit 1/{s/exit 1/#exit 1\n        echo "skip this"/}' ${DORIS_BE_SCRIPT}
+sed -i '/Please set the maximum number of open file descriptors/,/exit 1/{s/exit 1/#exit 1\n        echo "skip this"/}' ${DORIS_BE_SCRIPT}
+
+# start doris fe and be
+${DORIS_FE_HOME}/bin/start_fe.sh --daemon
+${DORIS_BE_HOME}/bin/start_be.sh --daemon
+
+# wait for the fe to start
+while true; do
```

Comment:
Should we include the maximum duration here, I'm afraid that the docker image will hang here forever.

Reply:
Good suggestion, I've updated.